linode/networkreservedips/datasource_list_schema.go

```diff
@@ -0,0 +1,47 @@
+package networkreservedips
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+type ReservedIPObject struct {
```

Comment:
Missing `vpc_nat_1_1`?

Reply:
I followed the schema of the closest existing resource - linode_networking_ip. We can still add this. Is it always null for reserved IPs? I see that when I reserve an IP address the vpc_nat_1_1 value is null. Need further clarification.